ignite/handlers/fbresearch_logger.py

```diff
@@ -1,18 +1,20 @@
 """FBResearch logger and its helper handlers."""
 
 import datetime
-from typing import Any, Optional
-
-# from typing import Any, Dict, Optional, Union
+from typing import Any, Callable, List, Optional
 
 import torch
 
+from ignite import utils
 from ignite.engine import Engine, Events
 from ignite.handlers import Timer
+from ignite.handlers.utils import global_step_from_engine  # noqa
```

Comment:
nit: we can remove this as there is no way of using `global_step_from_engine` with this logger